Add warning about Brave's "referral code" system (#2271)
privacyguides/privacyguides.org: docs/desktop-browsers.md

    1. We advise against using the Flatpak version of Brave, as it replaces Chromium's sandbox with Flatpak's, which is less effective. Additionally, the package is not maintained by Brave Software, Inc.

> !!! warning

Reviewer: > I noticed that Brave on macOS is distributed as pkg file, rather than a plain .app in a disk image. This is uncommon. As far as I know the other browser to do this is Microsoft Edge, which used a pkg installer to register its telemetry agent to launch at boot without even opening the browser.

To be honest, this whole .pkg nonsense is avoidable on macOS to begin with:

```suggestion
**macOS users:** The installer for Brave on their website is a `.pkg` installer which requires admin privileges to run. As an alternative, you can download the latest `Brave-Browser-universal.dmg` file from their [GitHub releases](https://github.com/brave/brave-browser/releases/latest) page, which provides a traditional "drag to Applications folder" install.

!!! warning
```

> !!! warning
>     Brave includes a 'promo code' in downloads from the Brave website, which is used to track which source the browser was downloaded from.
> 

Reviewer: ```suggestion
!!! warning

    Brave includes a 'promo code' in downloads from the Brave website, which is used to track which source the browser was downloaded from.
```

can we also recommend ways to avoid this? can you just rename the installer before opening it?

>     Brave includes a 'promo code' in downloads from the Brave website, which is used to track which source the browser was downloaded from.

Reviewer: ```suggestion
    Brave adds a "[referral code](https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes)" to the file name in downloads from the Brave website, which is used to track which source the browser was downloaded from, for example `BRV002` in a download named `Brave-Browser-BRV002.pkg`. The installer will then ping Brave's server with the referral code at the end of the installation process. If you're concerned about this, you can rename the installer file before opening it.
```
